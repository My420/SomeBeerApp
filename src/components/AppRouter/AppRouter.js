import React from 'react';
import { HashRouter } from 'react-router-dom';
import App from '../App/App';

const AppRouter = () => {
  // eslint-disable-next-line no-console
  console.log('render ===== AppRouter');
  return (
    <HashRouter>
      <App />
    </HashRouter>
  );
};
export default AppRouter;
