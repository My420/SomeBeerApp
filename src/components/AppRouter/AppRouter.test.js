import React from 'react';
import { shallow } from 'enzyme';
import AppRouter from './AppRouter';

describe('Test <AppRouter/> component', () => {
  const wrapper = shallow(<AppRouter />);

  test('should correct render component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
