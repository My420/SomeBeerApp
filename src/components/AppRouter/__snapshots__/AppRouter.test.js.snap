// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <AppRouter/> component should correct render component 1`] = `
<HashRouter>
  <ResizeProvider>
    <App />
  </ResizeProvider>
</HashRouter>
`;
